server/src/main/java/org/candlepin/guice/CandlepinFilterModule.java

```diff
@@ -57,6 +58,7 @@ protected void configureServlets() {
 
         filterRegex(regex).through(CandlepinScopeFilter.class);
         filterRegex(regex).through(CandlepinPersistFilter.class);
+//        filterRegex(noStatusResource).through(CandlepinSuspendModeFilter.class);
```

Comment:
This is leftover from the former filter implementation. Initially the filter to enter the suspend mode was implemented as Servlet Filter. But I have recently changed this to be a RestEasy filter. RestEasy filter is better because then we can utilize our, already in place, exception mappers and return uniform errors to the client